### PR TITLE
Read now button

### PR DESCRIPTION
**Pull request by ihatetoast**
Wee button made to take reading leader from the details page to the read page. pls check Router and Link. I chose the .button over .button-primary. not sure if you wanted plain or fancy.

### CODE REVIEW

**Comment by alarner** [comment]
We probably don't need a separate component for this because it's so simple. Can you just put that link in the details page"

File: public/scripts/components/pages/Details.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
 import React from 'react';
+import ReadNowButton from './../sub-components/ReadNowButton';
 
 export default React.createClass({
 	render: function() {
+		console.log(this.props.storyId);
```
Comment (alarner) [inline public/scripts/components/pages/Details.js]: Please remove console.log

**Comment by willcook4code** [comment]
👍

File: public/scripts/components/sub-components/ReadNowButton.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+// export default React.createClass({
+// 	render: function() {
+// 		return (
+				
+
+// 		);
+// 	}
+// });
+
```
Comment (alarner) [inline public/scripts/components/sub-components/ReadNowButton.js]: Please delete this file.

**Comment by MHG16** [comment]
👍

**Comment by joshwerner85** [comment]
👍

File: public/scripts/components/sub-components/Read_Now_Button.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+	}
+});
+
+//ROUTER
+// situation: path will be determined by the index of the story we are on now, not just 
+// 			any story. so not a fetch/get thing. 
+			
+	<Router history={browserHistory}>
+			<Route path='/stories/:storyId/read' component={Dontknowyet}</Route>
+	</Router>
```
Comment (alarner) [inline public/scripts/components/sub-components/Read_Now_Button.js]: Please delete this file

**Comment by alarner** [comment]
👍

**Comment by joshwerner85** [comment]
👍

**Comment by MHG16** [comment]
👍

**Pull request merged**